src/zigbee2mqtt/zigbee2mqtt-property.ts

```diff
@@ -0,0 +1,246 @@
+import Color from 'color';
+import { Property } from 'gateway-addon';
+import {
+  PropertyValue,
+  PropertyValueType,
+  Property as PropertySchema,
+} from 'gateway-addon/lib/schema';
+import { Expos } from './zigbee2mqtt-adapter';
+import { Zigbee2MqttDevice } from './zigbee2mqtt-device';
+import mqtt from 'mqtt';
+
+export const WRITE_BIT = 0b010;
+
+export function parseType(type?: string): PropertyValueType {
```

Comment:
This is duplicated in `zigbee2mqtt-device.ts` (although that one is missing `'binary'`).

Reply:
done